### PR TITLE
[#420] Use couchdb:2.1.1 for x86 and treehouses/rpi-couchdb:2.1.1 for arm image

### PR DESCRIPTION
**Pull request by mappuji**
Solve #420

### CODE REVIEW

File: docker/planet.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ services:
   couchdb:
     expose:
       - 5984
-    image: klaemo/couchdb
+    image: couchdb:2.1.1
```
Comment (lmmrssa) [inline docker/planet.yml]: @dogi if we want the latest of couchdb all time then we can just put couchdb on all places. Is that what you would prefer?

Comment (mappuji) [reply docker/planet.yml]: I guess the `klaemo/couchdb` version is not creating the `2.1+` version and moved to `couchdb` namespace

**Comment by mappuji** [comment]
Thanks @dogi

File: docker/planet.yml (hunk 1 above)
Comment (lmmrssa) [reply docker/planet.yml]: @mappuji you are right about the version. I was just pointing shall we do fix version or keep it as latest.

**Comment by dogi** [comment]
@lmmrssa and @xyb994 please test new Vagrantfile and template https://app.vagrantup.com/ole/boxes/stretch64/versions/0.3.2

**Comment by dogi** [comment]
```
    planet: 976efcf7d2516d3de9092c9d348d8f0cdddc3548e1372b07fb4c9e3daa689f51
    planet: { FetchError: request to http://localhost:5984/_membership failed, reason: read ECONNRESET
    planet:     at ClientRequest.<anonymous> (/home/vagrant/add-cors-to-couchdb/node_modules/node-fetch/index.js:133:11)
    planet:     at emitOne (events.js:96:13)
    planet:     at ClientRequest.emit (events.js:188:7)
    planet:     at Socket.socketErrorListener (_http_client.js:310:9)
    planet:     at emitOne (events.js:96:13)
    planet:     at Socket.emit (events.js:188:7)
    planet:     at emitErrorNT (net.js:1296:8)
    planet:     at _combinedTickCallback (internal/process/next_tick.js:80:11)
    planet:     at process._tickCallback (internal/process/next_tick.js:104:9)
    planet:   name: 'FetchError',
    planet:   message: 'request to http://localhost:5984/_membership failed, reason: read ECONNRESET',
    planet:   type: 'system',
    planet:   errno: 'ECONNRESET',
    planet:   code: 'ECONNRESET' }
```

**Comment by dogi** [comment]
twice
```
    planet: npm
    planet:  
    planet: WARN
    planet:  
    planet: deprecated
    planet:  minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
```

**Comment by dogi** [comment]
```
Your global Angular CLI version (1.7.3) is greater than your local
version (1.7.0). The local Angular CLI version is used.

To disable this warning use "ng set --global warnings.versionMismatch=false".
```

**Pull request merged**